README.md

```diff
@@ -14,178 +14,6 @@ If you need assistance or want to help, reach out to the [`bevy_cli` working gro
 [`bevy_cli` working group channel]: https://discord.com/channels/691052431525675048/1278871953721262090
 [Bevy Discord]: https://discord.gg/bevy
 
-## Installation
```

Comment:
Personally, I'd like to keep the installation instructions and quick start guide in the readme.
I like when you can click on the repo link and get started immediately without consulting external documentation.
The book can then contain the details on every command, configuration options, etc.

It's a bit of duplication though.

What do you think?

Reply:
I agree, it's important enough that we should take advantage of the visibility boost `README.md` provides.